src/documentation/wiki/breakout-deck.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Breakout deck
```

Comment:
Perhaps change the title to 'Breakout deck technical details'. The problem is now that the id_links function will render this on the product page as 'Breakout deck' on the product page which is not very subscriptive

Reply:
Agreed, updated.